File: cognizent_INT/Fibbonachhi.cs
using System;
public class Fibbonachhi
{
      public static void Main(String[] args)
      {
            Console.WriteLine("Enter Limit: ");
            int num=int.Parse(Console.ReadLine());
            int a=0,b=1;
            int c;
            for(int i=0;i<num;i++)
            {
                  Console.Write(a+" ");
                  c=a+b;
                  a=b;
                  b=c;
            }
            Console.WriteLine();
      }
}

File: cognizent_INT/Fibbonachhi.java
import java.util.Scanner;

public class Fibbonachhi {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter Limit: ");
        int num = Integer.parseInt(scanner.nextLine());
        int a = 0, b = 1;
        int c;
        for (int i = 0; i < num; i++) {
            System.out.print(a + " ");
            c = a + b;
            a = b;
            b = c;
        }
        System.out.println();
    }
}
